ios/Podfile

```diff
@@ -36,6 +36,7 @@ target 'FriendlyPlans' do
   pod 'Firebase/Core', '~> 5.20.2'
   pod 'Firebase/Firestore', '~> 5.20.2'
   pod 'Firebase/Auth', '~> 5.20.2'
+  pod 'Firebase/Storage'
```

Comment:
I'm surprised it wasn't connected for iOS. 
Can we lock the version like with the rest of Firebase libraries?